keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/ExportController.kt

```diff
@@ -61,15 +61,14 @@ class ExportController(
    */
   @GetMapping(
     path = ["/{cloudProvider}/{account}/{type}/{name}"],
-    params = ["serviceAccount"],
-    produces = [MediaType.APPLICATION_JSON_VALUE, APPLICATION_YAML_VALUE]
+    produces = [APPLICATION_JSON_VALUE, APPLICATION_YAML_VALUE]
   )
   fun get(
     @PathVariable("cloudProvider") cloudProvider: String,
     @PathVariable("account") account: String,
     @PathVariable("type") type: String,
     @PathVariable("name") name: String,
-    @RequestParam("serviceAccount") serviceAccount: String
+    @RequestHeader("X-SPINNAKER-USER") user: String
```

Comment:
Exporting no longer requires a service account (as it's not included in the exported resource). Instead we use the Spinnaker user header when we need to authenticate with CloudDriver.